pkg/operator/controller/controller_dns_daemonset.go

```diff
@@ -28,7 +27,11 @@ func (r *reconciler) ensureDNSDaemonSet(dns *operatorv1.DNS) (bool, *appsv1.Daem
 	if err != nil {
 		return false, nil, err
 	}
-	desired, err := desiredDNSDaemonSet(dns, r.CoreDNSImage, r.KubeRBACProxyImage)
+	cmMap, err := caBundleRevisionMap(r.client, dns)
```

Comment:
`caBundleRevisionMap` potentially does a lot of `Get`s.  Can you call `caBundleRevisionMap` in `ensureDNS` and then pass the result to `ensureDNSDaemonSet` and to `ensureDNSConfigMap` to avoid calling `caBundleRevisionMap` twice for every reconciliation?

Reply:
Done.